src/js/control-bar/mute-toggle.js

```diff
@@ -92,7 +95,7 @@ class MuteToggle extends Button {
     // Don't rewrite the button text if the actual text doesn't change.
     // This causes unnecessary and confusing information for screen reader users.
     // This check is needed because this function gets called every time the volume level is changed.
-    const toMute = this.player_.muted() ? 'Unmute' : 'Mute';
+    const toMute = this.player_.muted() ? 'Mute Toggle Unmute' : 'Mute Toggle Mute';
```

Comment:
this would change our localizations, I think it would be better not to change it, if we don't need to.